drivers/adc/adc_stm32.c

```diff
@@ -280,16 +288,15 @@ static void adc_stm32_start_conversion(const struct device *dev)
 #if !defined(CONFIG_SOC_SERIES_STM32F2X) && \
 	!defined(CONFIG_SOC_SERIES_STM32F4X) && \
 	!defined(CONFIG_SOC_SERIES_STM32F7X) && \
-	!defined(CONFIG_SOC_SERIES_STM32F1X) && \
+	!defined(CONFIG_SOC_SERIES_STM32F1X) && !defined(STM32F3X_ADC_V2_5) && \
```

Comment:
Please follow the existing driver style and use one operand per line.
Apply everywhere